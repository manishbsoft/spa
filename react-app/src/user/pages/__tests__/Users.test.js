import React from 'react';
import Users from '../Users';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


configure({ adapter: new Adapter() });

describe("Users Component", () => {
  
  const wrapper = shallow(<Users />);

  it("should render Users Component", () => {  
    expect(wrapper).toMatchSnapshot();
  });
});