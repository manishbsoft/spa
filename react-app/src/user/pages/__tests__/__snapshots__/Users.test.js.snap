// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users Component should render Users Component 1`] = `ShallowWrapper {}`;
